### PR TITLE
Updated readme.md & nuget versioning scheme

### PR DESCRIPTION
**Pull request by mlatoszek**
Updated readme file
Added status badges

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,5 @@
+![Automation Library CI/CD](https://github.com/kmdlogic/kmd-logic-gateway-automation/workflows/Automation%20Library%20CI/CD/badge.svg?branch=master)
+![CLI Tool CI/CD](https://github.com/kmdlogic/kmd-logic-gateway-automation/workflows/CLI%20Tool%20CI/CD/badge.svg?branch=master)
+
 # kmd-logic-gateway-automation
-Kmd loci
+Kmd Logic gateway automation
```
Comment (jaceks2106) [inline README.md:5]: Upper case please

**Review by jaceks2106 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (jaceks2106) [reply README.md]: Ahhh. whatever. It must be totally rewritten :D

Comment (mlatoszek) [reply README.md]: Already fixed :P But please see status badges! :)

**Review by jaceks2106 — APPROVED** [review]

File: Directory.Build.props [hunk 2]
```diff
@@ -17,7 +17,10 @@
 		<PackageLicenseExpression>MIT</PackageLicenseExpression>
 		<RepositoryUrl>https://github.com/kmdlogic/kmd-logic-gateway-automation</RepositoryUrl>
 		<RepositoryType>git</RepositoryType>
-		<GenerateAssemblyVersionAttribute>false</GenerateAssemblyVersionAttribute>
+
+		<VersionPrefix Condition=" '$(VersionPrefix)' == '' ">0.1.0</VersionPrefix>
```
Comment (jaceks2106) [inline Directory.Build.props:21]: Verify if `VersionSuffix` is still required in `src/Kmd.Logic.Gateway.Automation.Tool/Kmd.Logic.Gateway.Automation.Tool.csproj` and library as well

Comment (mlatoszek) [reply Directory.Build.props:21]: Removed as agreed on Teams :)

**Review by jaceks2106 — APPROVED** [review]

**Pull request merged**